test/web-platform-tests/to-run.yaml

```diff
@@ -498,19 +495,13 @@ button.html: [fail, Validation is not implemented]
 checkbox.html: [fail, Events weirdness]
 datetime-local.html: [fail, min/max constraints not implemented]
 datetime.html: [fail, step is not implemented]
-email.html: [fail, Validation is not implemented]
 hidden.html: [fail, list/step/validation/valueAsDate/valueAsNumber not implemented]
```

Comment:
If the validation-part of this test is passing now, you can remove "validation" from the description here.